source/api_documentation/transactions_api/_intro.rst

```diff
@@ -59,7 +59,9 @@ The API takes the following optional query parameters:
     - Filters for the type of transaction. Valid inputs are Call, PostCallEvent, Sale, or Signal. Sale maps to the Reported Conversion type.
 
 In order to ensure that all transactions are returned when using the from= and to= date query parameters,
-you should store the last transaction id you have downloaded and pass it as the start_after_transaction_id to the next request.
+you should store the last transaction id you have downloaded and pass it as the start_after_transaction_id to the next request *so long as you do not change the date range*.
+Once you update the date range, the best practice is to omit the start_after_transaction_id param for the first request, then use the last transaction id you have downloaded until the date range is complete.
+This is because using from and to params sorts transactions in relation to the call start time, not the transaction time.
```

Comment:
Note: based on the prior text that was here v. how it's displayed, this will all be one paragraph: https://developers.invoca.net/en/2022-08-01/api_documentation/transactions_api/network_user.html

Reply:
This definitely helps clarify how to use these params